tests/test_moving_eddies.py

```diff
@@ -96,6 +96,39 @@ def moving_eddies_example(grid, npart=2, mode='jit', verbose=False,
 
 
 @pytest.mark.parametrize('mode', ['scipy', 'jit'])
+def moving_eddies_fwdbwd(mode, npart=2):
```

Comment:
The new test isn't being picked up by pytest. According to conventions it should be called `test_moving_eddies_fwdbwd` to add it to the automated test suite.